components/mediation-admin/org.wso2.carbon.message.processor/src/main/java/org/wso2/carbon/message/processor/service/MessageProcessorAdminService.java

```diff
@@ -755,4 +762,110 @@ public boolean validateAxis2ClientRepo(String input) throws AxisFault {
         }
     }
 
+
+    /**
+     * Gets the message from the associated queue
+     *
+     * @param processorName Message Processor name
+     * @return <code>msg</code> Returns message received from the queue as a string
+     * @throws Exception
+     */
+
+    public String getMessage(String processorName) {
```

Comment:
We should not have a line space between the method definition and the JAVA doc comment.